tests/suites/test_suite_psa_crypto.function

```diff
@@ -1287,7 +1375,16 @@ void aead_key_policy( int policy_usage_arg,
     else
         TEST_EQUAL( status, expected_status );
 
+    /* Encrypt check, multi-part */
+    PSA_ASSERT( psa_aead_abort( &operation ) );
+    status = psa_aead_decrypt_setup( &operation, key, exercise_alg );
+    if( ( policy_usage & PSA_KEY_USAGE_DECRYPT ) == 0 )
+        TEST_EQUAL( status, PSA_ERROR_NOT_PERMITTED );
+    else
+        TEST_EQUAL( status, expected_status );
```

Comment:
I removed the
```
else if( expected_status == PSA_SUCCESS )
        TEST_EQUAL( status, PSA_ERROR_INVALID_SIGNATURE );
```
on purpose since this causes :
```
PSA key policy: AEAD, encrypt | decrypt ........................... FAILED
  status == PSA_ERROR_INVALID_SIGNATURE
  at line 1384, suites/test_suite_psa_crypto.function
  lhs = 0x0000000000000000 = 0
  rhs = 0xffffffffffffff6b = -149
```
```
PSA key policy: AEAD, decrypt but not encrypt ..................... FAILED
  status == PSA_ERROR_INVALID_SIGNATURE
  at line 1384, suites/test_suite_psa_crypto.function
  lhs = 0x0000000000000000 = 0
  rhs = 0xffffffffffffff6b = -149
```
```
PSA key policy: AEAD, tag length > min-length policy, CCM ......... FAILED
  status == PSA_ERROR_INVALID_SIGNATURE
  at line 1384, suites/test_suite_psa_crypto.function
  lhs = 0x0000000000000000 = 0
  rhs = 0xffffffffffffff6b = -149
```
```
PSA key policy: AEAD, tag length = min-length policy, CCM ......... FAILED
  status == PSA_ERROR_INVALID_SIGNATURE
  at line 1384, suites/test_suite_psa_crypto.function
  lhs = 0x0000000000000000 = 0
  rhs = 0xffffffffffffff6b = -149
```
```
PSA key policy: AEAD, tag length > min-length policy, GCM ......... FAILED
  status == PSA_ERROR_INVALID_SIGNATURE
  at line 1384, suites/test_suite_psa_crypto.function
  lhs = 0x0000000000000000 = 0
  rhs = 0xffffffffffffff6b = -149
```
```
PSA key policy: AEAD, tag length = min-length policy, GCM ......... FAILED
  status == PSA_ERROR_INVALID_SIGNATURE
  at line 1384, suites/test_suite_psa_crypto.function
  lhs = 0x0000000000000000 = 0
  rhs = 0xffffffffffffff6b = -149
```
```
PSA key policy: AEAD, default tag length > min-length policy ...... FAILED
  status == PSA_ERROR_INVALID_SIGNATURE
  at line 1384, suites/test_suite_psa_crypto.function
  lhs = 0x0000000000000000 = 0
  rhs = 0xffffffffffffff6b = -149
PSA key policy: AEAD, default tag length = min-length policy ...... FAILED
  status == PSA_ERROR_INVALID_SIGNATURE
  at line 1384, suites/test_suite_psa_crypto.function
  lhs = 0x0000000000000000 = 0
  rhs = 0xffffffffffffff6b = -149
```
```
PSA key policy: AEAD, tag length = exact-length policy ............ FAILED
  status == PSA_ERROR_INVALID_SIGNATURE
  at line 1384, suites/test_suite_psa_crypto.function
  lhs = 0x0000000000000000 = 0
  rhs = 0xffffffffffffff6b = -149
```

Reply:
Looking at https://armmbed.github.io/mbed-crypto/html/api/ops/aead.html?highlight=psa_aead_decrypt_setup#c.psa_aead_decrypt_setup `psa_aead_decrypt_setup` can't return `PSA_ERROR_INVALID_SIGNATURE`